main.js

```diff
@@ -0,0 +1,180 @@
+let rockImage = document.querySelector("#rock");
+let paperImage = document.querySelector("#paper");
+let scissorsImage = document.querySelector("#scissors");
+let you = document.querySelectorAll(`.you`);
+let textButton = document.querySelector(`#text-button`);
+let inputtedWord = document.getElementById("changeText");
+let playAgain = document.querySelector("#play-again");
+
+// Player Clicks on Image which is stored in event
+// When player clicks on choice, Computer Picks their Random Choice
+
+function computerPicks() {
+  let choice = ["Wrestler", "Sumo", "Samurai"];
+
+  let random = Math.floor(Math.random() * choice.length);
+
+  computerChoice = (random, choice[random]);
+
+  let computerMove = computerChoice;
+
+  return computerMove;
+}
+
+// Compare Player Choice to Computer
+
+// Rock
+
+function playerRock(event) {
+  const audioSumo = new Audio("growl.wav");
+  audioSumo.play();
+  playerChoiceText.classList.add("largerText");
+  playerChoiceText.innerText = "You picked Rocky the Wrestler!";
+  playerMove = "Wrestler";
+  computerMove = computerPicks();
+  if (computerMove === "Wrestler") {
+    drawIncrease();
+  } else if (computerMove === "Samurai") {
+    playerScoreIncrease();
+  } else if (computerMove === "Sumo") {
+    lossIncrease();
+  }
+  gamesScoreIncrease();
+}
+
+rockImage.addEventListener("click", playerRock);
+
+//paper
+
+function playerPaper(event) {
+  const audioSumo = new Audio("sumo.wav");
+  audioSumo.play();
+  playerChoiceText.innerText = "You picked Sumo!";
+
+  playerMove = "Sumo";
+  computerMove = computerPicks();
+  if (computerMove === "Sumo") {
+    drawIncrease();
+  } else if (computerMove === "Wrestler") {
+    playerScoreIncrease();
+  } else if (computerMove === "Samurai") {
+    lossIncrease();
+  }
+
+  gamesScoreIncrease();
+}
+
+paperImage.addEventListener("click", playerPaper);
+
+//scissors
+
+function playerScissors(event) {
+  const audioSamurai = new Audio("sword.mp3");
+  audioSamurai.play();
+  playerChoiceText.innerText = "You picked Samurai!";
+  playerMove = "Samurai";
+  computerMove = computerPicks();
+  if (computerMove === "Samurai") {
+    drawIncrease();
+  } else if (computerMove === "Sumo") {
+    playerScoreIncrease();
+  } else if (computerMove === "Wrestler") {
+    lossIncrease();
+  }
+
+  gamesScoreIncrease();
+}
+
+scissorsImage.addEventListener("click", playerScissors);
+
+// Scores
+
+function gamesScoreIncrease() {
+  let gamesScore = document.querySelector("#score-games");
+  result = gamesPlayed++;
+  gamesScore.innerText = result;
+}
+
+function playerScoreIncrease() {
+  let playerScore = document.querySelector("#score-won");
+  result = playerWins++;
+  playerScore.innerText = result;
+  setTimeout(() => {
+    const audioWin = new Audio("win.mp3");
+    audioWin.play();
+  }, 500);
+  displayComputerChoice.innerText =
+    "The Master picked " + computerMove + "!" + "\nVICTORY IS YOURS!!!!!";
```

Comment:
Remember that you can use string interpolation to inject variables into strings as well. That way it's a little cleaner/closer to how we read actual text, and you don't have to worry about adding spaces in the right places around the plusses, etc.

Example: 
```js
`The Master picked ${computerMove}! \nVICTORY IS YOURS!!!!!`
```